### PR TITLE
Open Source Foundations

### PR DESCRIPTION
**Pull request by BrittanyIstenes**
Added in blog for why working in foundations is a good thing

### CODE REVIEW

**Comment by robmoffat** [comment]
Hi @BrittanyIstenes,

Added the metadata and merged with the existing article introducing foundations.  Added links to all the things you mention.

https://deploy-preview-159--open-source-readiness.netlify.app/docs/bok/Activities/Level-4/Foundations

File: docs/bok/Activities/Level-4/Foundations.md [hunk 1]
```diff
@@ -9,23 +9,72 @@ sidebar_position: 5
...
+
+Open source foundations play a vital role in securing resources and funding for open source projects. They often act as intermediaries between individual contributors, corporate sponsors, and grant organizations, ensuring the financial sustainability of projects. By providing fiscal and administrative support, foundations enable developers to focus on the project's technical aspects and community engagement, rather than solely on fundraising.
+
+**See: **
+ - [Dependency Risk](../../Risks/Dependency-Risk)
+
+## Prominent Open Source Foundations
+
+- **[The Linux Foundation](https://www.linuxfoundation.org/)**: Known for its work on the Linux kernel, the Linux Foundation supports numerous critical open source projects, such as [Node.js](https://nodejs.org), [Hyperledger](https://www.hyperledger.org) and of course the [Linux Kernel](https://kernel.org/category/about.html)
+- **[Apache Software Foundation (ASF)](https://community.apache.org/)**: ASF oversees projects like [Apache HTTP Server](https://httpd.apache.org), [Hadoop](https://hadoop.apache.org), and [Kafka](https://kafka.apache.org), promoting open development and community-driven decision-making.
```
Comment (tsteenbe) [inline docs/bok/Activities/Level-4/Foundations.md:44]: Recommend we add Eclipse Foundation to the list as well, most FS companies use Java and thereby use artifacts from Eclipse project. As examples I recommend adding https://adoptium.net/, https://projects.eclipse.org/projects/oniro.

File: docs/bok/Activities/Level-4/Foundations.md [hunk 2]
```diff
@@ -9,23 +9,72 @@ sidebar_position: 5
...
+- **[Open Source Initiative (OSI)](https://opensource.org/)**: OSI serves as a steward of the Open Source Definition and approves licenses as compliant with open source principles, promoting openness and collaboration.
+- **[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/)**: CNCF, under the Linux Foundation, sustains projects like [Kubernetes](https://kubernetes.io), [Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/), and [Envoy](https://www.cncf.io/projects/envoy/), fostering the cloud-native ecosystem.
```
Comment (tsteenbe) [inline docs/bok/Activities/Level-4/Foundations.md:46]: Should we add a section that briefly explain the difference between foundations in operating modus. For example when open-sourcing or moving a project under a foundation - with the Linux Foundation is kind-a-like a start up environment where each project get lots of freedom to develop themselves how the participants see fit. Whereas with the Eclipse it's more open-sourcing with built-in guardrails. Apache is more community driven/consensus. Each approach has its pros and cons but most people/companies are not familiar with different ways of working between foundation.

**Review by tsteenbe — DISMISSED** [review]

**Comment by pholleran** [comment]
One of the things I feel is missing from this doc is an explanation as to why a firm may seek to propose a project to a foundation. How might they evaluate the decision?

It comes with some positives and negatives:

Positives:
* Potentially wider adoption and participation
* Shared administrative load (maintainership)

"Negatives":
* Firm gives up "singular" control
  * Though, is this really a negative as it may give others pause before participating

File: docs/bok/Activities/Level-4/Foundations.md [hunk 3]
```diff
@@ -9,23 +9,72 @@ sidebar_position: 5
...
+
+Open source foundations offer legal frameworks and support to ensure projects are compliant with licenses, intellectual property rights, and patents. They help navigate the complexities of licensing, copyrights, and trademarks, reducing the legal risks associated with using and contributing to open source projects. By providing legal clarity, foundations enable developers and organizations to confidently contribute to and build upon open source software.
+
+**See: ** 
+ - [Anti-Trust Regulations](../../Regulations/Anti-Trust)
+ - [Reputational Risk](../../Risks/Reputational-Risk)
+
+### 3. Funding and Sustainability
+
+Open source foundations play a vital role in securing resources and funding for open source projects. They often act as intermediaries between individual contributors, corporate sponsors, and grant organizations, ensuring the financial sustainability of projects. By providing fiscal and administrative support, foundations enable developers to focus on the project's technical aspects and community engagement, rather than solely on fundraising.
```
Comment (coopernetes) [inline docs/bok/Activities/Level-4/Foundations.md:36]: I would also expand on the administration of funding as another large benefit of working within a foundation for an open source project. Just a brief blurp on how a foundation will manage the taxation, accounting and regulatory requirements that comes from being a non-profit such as [501(c)(3)](https://www.irs.gov/charities-non-profits/charitable-organizations/exemption-requirements-501c3-organizations). This assists project maintainers and stewards with administering funds required to run a larger projects that may have infrastructure fees, travel, salary, ancillary expenses, etc.

File: docs/bok/Activities/Level-4/Foundations.md [hunk 4]
```diff
@@ -9,23 +9,72 @@ sidebar_position: 5
...
 
+(this part to be completed)
+
 Sponsoring third-party projects or individuals
 Sponsoring own organization's open source projects or individual members
 Sponsoring open source organizations
 Sponsoring events
 
  - Under what conditions should the code be held by the bank's own organisation, vs. contributed to a foundation?
+
```
[the remark this answers is not in the document]
Comment (BrittanyIstenes) [reply docs/bok/Activities/Level-4/Foundations.md:76]: @robmoffat can you list out some resources you want me to add?

File: docs/bok/Activities/Level-4/Foundations.md (hunk 1 above)
Comment (BrittanyIstenes) [reply docs/bok/Activities/Level-4/Foundations.md:44]: This will be added

File: docs/bok/Activities/Level-4/Foundations.md (hunk 2 above)
Comment (BrittanyIstenes) [reply docs/bok/Activities/Level-4/Foundations.md:46]: @tsteenbe  - do you want to collaborate on a brief synopsis on this?

**Comment by BrittanyIstenes** [comment]
We need further discussion for the deeper dives from Peter, Phil and Thomas so that I can interpret more directly the needs and the why.   It is challenging to go through and edit the document directly without more discussion.

File: docs/bok/Activities/Level-4/Foundations.md (hunk 3 above)
Comment (coopernetes) [reply docs/bok/Activities/Level-4/Foundations.md:36]: After thinking on this @BrittanyIstenes , I think your original paragraph captures the essence of the benefits of having a foundation handling the funding already. My above comment is a bit too "in the weeds" on second glance. Feel free to disregard.

**Comment by robmoffat** [comment]
@psmulovics @tsteenbe @pholleran thanks for all your comments.  @BrittanyIstenes and I are just reviewing them but it would be really useful to have some suggested text from you.  e.g. @tsteenbe you talk about the difference between foundations, which is useful, but can you give some text that we can include?  Ideally some references too.  

ty!

**Comment by robmoffat** [comment]
Added further reading section

**Review by robmoffat — APPROVED** [review]
please add the bit on sponsorship then let's merge

**Pull request merged**